lib/ClangImporter/ImportMacro.cpp

```diff
@@ -299,6 +293,47 @@ static Optional<clang::QualType> builtinTypeForToken(const clang::Token &tok,
   }
 }
 
+static Optional<std::pair<llvm::APSInt, Type>>
+  getIntegerConstantForMacroToken(ClangImporter::Implementation &impl,
+  	                              DeclContext *DC,
+    	                            const clang::Token &token) {
```

Comment:
Some tabs crept in here instead of spaces. We would usually go with

```c++
static Optional<std::pair<llvm::APSInt, Type>>
getIntegerConstantForMacroToken(ClangImporter::Implementation &impl,
                                DeclContext *DC,
                                const clang::Token &token) {
```